api/managementpb/rds.proto

```diff
@@ -114,6 +114,8 @@ message AddRDSRequest {
   string agent_password = 29;
   // Database name.
   string database = 30;
+  // Limit of databases for Auto-discovery.
```

Comment:
```suggestion
  // Limit of databases for auto-discovery.
```

Reply:
@ademidoff Based on https://perconacorp.slack.com/archives/C03FB1CAZH9/p1699281782390989?thread_ts=1699280560.488959&cid=C03FB1CAZH9 I changed it everywhere to have same "naming". 

Are we going to have "Auto-discovery" in UI and CLI descriptions and in comments "auto-discovery"?

Reply:
It's not a proper name and not an abbreviation to deserve a capital letter. When we start the sentence, we say "Auto-discovery", otherwise it should be "auto-discovery". I see no indication in that slack thread that it should be capital "A".

Reply:
Catalina literally written it with "A". Changed.